pkg/server/server.go

```diff
@@ -288,6 +288,20 @@ func (s *Server) CreateRequest(c *gin.Context, user *models.UserCredentials) {
 		return
 	}
 
+	if user.UnverifiedEmail != "" {
```

Comment:
Can we have this condition only for admin-created accounts (Core) and not others (Portal)?

Reply:
Yes, This function is called by admin API(Core) only. For signup (Portal) we have a different API in which this check is not there.

Reply:
ok